src/services/shell.service.js

```diff
@@ -16,7 +16,7 @@ export const openProjectInEditor = (project: Project) =>
 
 export const getNodeJsVersion = () =>
   new Promise(resolve =>
-    exec('node -v', (error, stdout) => {
+    exec('node -v', { env: window.process.env }, (error, stdout) => {
```

Comment:
I _think_ if you don't add the `encoding: 'utf8'` option, it defaults to `Buffer` and `trim()` will fail

Reply:
It's `utf8` encoded by default as mentioned in the [docs](https://nodejs.org/dist/latest-v8.x/docs/api/child_process.html#child_process_child_process_exec_command_options_callback).

Reply:
Oh, the test fails because of this line. 
Maybe adding `.toString().trim()` will fix this. I'll check this later today.